aws/resource_aws_codepipeline_test.go

```diff
@@ -378,6 +378,41 @@ func TestAccAWSCodePipeline_multiregion_ConvertSingleRegion(t *testing.T) {
 	})
 }
 
+func TestAccAWSCodePipelineWithNamespace(t *testing.T) {
+	if os.Getenv("GITHUB_TOKEN") == "" {
+		t.Skip("Environment variable GITHUB_TOKEN is not set")
+	}
+
+	var p1, p2 codepipeline.PipelineDeclaration
```

Comment:
Lint is failing because p2 is declared but never used

Reply:
Thanks. I'll remove that.